torch/csrc/jit/autodiff.cpp

```diff
@@ -676,7 +690,13 @@ static void deduplicateSizeCaptures(
     }
     if (usedOnlyInReverse(capture) && capture_set.count(node->input())) {
       WithInsertPoint insert_guard{*rev_info.reverse_block->nodes().begin()};
-      capture->replaceAllUsesWith(SymbolicVariable(node->input()).size());
+      auto size = SymbolicVariable(node->input()).size();
```

Comment:
Is `SymbolicVariable` necessary here?

Reply:
I don't think so; Mike should be removing these shortly!